### PR TITLE
Bohra v1.0.19

### PR DESCRIPTION
**Pull request by kristyhoran**
Bioconda requires reviews prior to merging pull-requests (PRs). To facilitate this, once your PR is passing tests and ready to be merged, please add the `please review & merge` label so other members of the bioconda community can have a look at your PR and either make suggestions or merge it. Note that if you are not already a member of the bioconda project (meaning that you can't add this label), please ping `@bioconda/core` so that your PR can be reviewed and merged (please note if you'd like to be added to the bioconda project). Please see https://github.com/bioconda/bioconda-recipes/issues/15332 for more details.

* [X] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/contributor/guidelines.html).
* [X] This PR adds a new recipe.
* [X] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/bohra/LICENSE.txt [hunk 1]
```diff
@@ -0,0 +1,60 @@
+for a work that has been modified or installed by the recipient, or for
```
Comment (dpryan79) [inline recipes/bohra/LICENSE.txt]: The first half of the license appears to be missing.

File: recipes/bohra/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+    - bohra --help
+
+about:
+  home: "https://github.com/kristyhoran/bohra"
+  license: "GNU General Public v3 (GPLv3)"
+  license_family: "GPL3"
+  license_file: "LICENSE.txt"
+  summary: "A bioinformatics pipeline for analysing short read Illumina data microbiological public health."
+  doc_url: ""
+  dev_url: ""
```
Comment (dpryan79) [inline recipes/bohra/meta.yaml]: Please remove `doc_url` and `dev_url`.

File: recipes/bohra/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+
+build:
+  noarch: generic
+  number: 0
+  entry_points:
+    - bohra=bohra.bohra:main
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
```
Comment (bgruening) [inline recipes/bohra/meta.yaml]: I think everything can be removed from host except pip and python.

File: recipes/bohra/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+    - shovill
+    - roary
+    - prokka
+    - kraken2
+    - abricate
+    - mlst
+    - iqtree
+    - seqtk
+    - snp-dists
+    - zlib
```
Comment (bgruening) [inline recipes/bohra/meta.yaml]: why is zlib here needed?

File: recipes/mlst/meta.yaml [hunk 5]
```diff
@@ -10,10 +10,17 @@ source:
   sha256: '{{ sha256 }}'
 
 build:
-  number: 0
+  number: 1
 
   noarch: generic
 requirements:
+  host:
```
Comment (bgruening) [inline recipes/mlst/meta.yaml]: any reason why this is now needed?

File: recipes/bohra/meta.yaml (hunk 3 above)
Comment (kristyhoran) [reply recipes/bohra/meta.yaml]: @bgruening The python dependencies are all required in host, but you were correct re the rest. I have made these changes and pushed them! Thank yo for the suggestion

File: recipes/bohra/meta.yaml (hunk 4 above)
Comment (bgruening) [reply recipes/bohra/meta.yaml]: ping

File: recipes/mlst/meta.yaml (hunk 5 above)
Comment (bgruening) [reply recipes/mlst/meta.yaml]: ping

File: recipes/bohra/meta.yaml (hunk 4 above)
Comment (kristyhoran) [reply recipes/bohra/meta.yaml]: @bgruening the zlib has now been removed from current recipe

File: recipes/mlst/meta.yaml (hunk 5 above)
Comment (kristyhoran) [reply recipes/mlst/meta.yaml]: @bgruening sorry to be dense, but I am not sure I understand the question. I have tested locally and it appears that the host is required with the python deps. And if I leave the noarch: generic out the build does not pass linting. Can you please provide some info re exactly what you are referring to?

Comment (kristyhoran) [reply recipes/mlst/meta.yaml]: @bgruening I was following the skeleton for recipes.

**Pull request merged**